### ISSUE TITLE
regression task

### ISSUE THREAD

**Issue by yuqinT10** [deepomicslab/GNNFrame#10]
Hello, I’m currently working with GCNFrame and wondering if it’s possible to modify the output layer to perform a regression task instead of a classification task？

**Comment by wangruohan111** [comment]
Yes, but GCNmodel.py should be modified to adapt to a regression task.
